### ISSUE TITLE
[QUESTION] Disable some sensors

### ISSUE THREAD

**Issue by lacrima08** [MaxWinterstein/toogoodtogo-ha-mqtt-bridge#151]
Hi,

I'm a new user of this addon and have a question. I'm sorry if it's a dumb question.

I have some favorites on my TGTG mobile app but don't want to monitor some stores in the add-on. Is there a way to do that, plz ?

Thanks

**Comment by MaxWinterstein** [comment]
Not really. I guess using two seperate accounts might be the best way for this.
